test/css/value/unicode.0.json

```diff
@@ -0,0 +1,92 @@
+{
+  "type": "value",
+  "content": [
+    {
+      "type": "ident",
+      "content": "U",
```

Comment:
Looking back at this, I have to agree that there should only be one ident of `U+0-7F` :+1: